code/modules/ninja/energy_katana.dm

```diff
@@ -73,9 +73,9 @@
 	..()
 
 /obj/item/energy_katana/Destroy()
+	. = ..()
 	QDEL_NULL(spark_system)
 	QDEL_NULL(jaunt)
-	return ..()
 
```

Comment:
destroy's parent call should almost always be done last, since otherwise the cleanup we do after it could cause more references to hang 

It's weird that you got the error you did

Reply:
I believe this make sense actually:
 * Since this is an item, parent delete will temporary-drop
 * The katana upon dropping will `jaunt.Remove(user)`
 * We just deleted `jaunt`, so obviously this errors out

I'd say the bigger question is how the loc loop would cause a dummy wielded katana to delete (???)